src/Catty/DataModel/Configuration/PenConfiguration.swift

```diff
@@ -22,7 +22,23 @@
 
 struct PenConfiguration {
     var penDown = false
-    var size = SpriteKitDefines.defaultPenSize
+    static let sizeConversionFactor = 0.634
+
+    private(set) var size: CGFloat = SpriteKitDefines.defaultCatrobatPenSize * CGFloat(PenConfiguration.sizeConversionFactor)
+
+    var catrobatSize: CGFloat {
+        set {
+            size = CGFloat(PenConfiguration.sizeConversionFactor) * newValue
+        }
+        get {
+            size / CGFloat(PenConfiguration.sizeConversionFactor)
+        }
+    }
+
     var color = SpriteKitDefines.defaultPenColor
     var previousPosition: CGPoint?
+
+    init() {
+        catrobatSize = SpriteKitDefines.defaultCatrobatPenSize
```

Comment:
I think this can now be removed since the default value is set directly at the `size` property.